autorest/codegen/__init__.py

```diff
@@ -111,6 +118,15 @@ def _build_exceptions_set(yaml_data: List[Dict[str, Any]]) -> Set[int]:
                     exceptions_set.add(id(exception["schema"]))
         return exceptions_set
 
+    @staticmethod
+    def _build_package_dependency() -> Dict[str, str]:
+        return {
+            "dependency_azure_mgmt_core": "azure-mgmt-core>=1.3.0,<2.0.0",
```

Comment:
small nit: i don't think we need this anymore now that setup,py is condensed into one file. But feel free to keep! (i know i suggested it in the first place 😄 )

Reply:
I think your first suggestion is better since we could manage these dependencies more conveniently here 😄